tests/kola/data/libtest.sh

```diff
@@ -81,3 +81,10 @@ assert_not_file_has_content_literal () {
     done
 }
 
+# Mount the EFI partition at a temporary location.
+tmpefimount=/tmp/efi
```

Comment:
I know this is just the test suite, but I try as a general rule to avoid predictably named temporary files.  We could have this code mirror the main logic and use e.g. `$(mktemp -d)` and "return" it by printing, so
callers do e.g.:
`tmpefimount=$(mount_tmp_efi)`
etc.

Not a blocker, just noting.